mattermost-webapp/en.json

```
@@ -3591,7 +3591,7 @@
   "multiselect.numGroupsRemaining": "Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {group} other {groups}}. ",
   "multiselect.numMembers": "{memberOptions, number} of {totalCount, number} members",
   "multiselect.numPeopleRemaining": "Use ↑↓ to browse, ↵ to select. You can add {num, number} more {num, plural, one {person} other {people}}. ",
-  "multiselect.numRemaining": "You can add {num, number} more",
+  "multiselect.numRemaining": "Up to {max} can be added at a time. You have {num, number} remaining.",
```

Yeah, this almost works with the corresponding code changes. Just one other thing is needed.
```suggestion
  "multiselect.numRemaining": "Up to {max, number} can be added at a time. You have {num, number} remaining.",
```
Max will never be 0, but if it was, the message would appear as "Up to  can be added at a time..." due to some quirks in JavaScript.